examples/tof-viewer/src/ADIMainWindow.cpp

```diff
@@ -581,6 +584,53 @@ void ADIMainWindow::showRecordMenu() {
     }
 }
 
+void ADIMainWindow::showSaveAdsdParamsMenu() {
+
+    char pathname[512];
```

Comment:
Is there a reason to use char array instead of std::string?
You can directly use saveConfigurationPath, thus avoiding doing a copy.

Reply:
imgui::inputtext need a char array to copy the text from UI.

Reply:
makes sense. Thanks!